frontends/p4/frontend.cpp

```diff
@@ -211,9 +211,12 @@ const IR::P4Program *FrontEnd::run(const CompilerOptions &options, const IR::P4P
         new SwitchAddDefault,
         new FrontEndDump(),  // used for testing the program at this point
         new RemoveAllUnusedDeclarations(*policy, true),
+        // Give each local declaration a unique internal name.
```

Comment:
Maybe SimplifyParsers should run that pass to be safe? Or we should add a note to that pass.

Reply:
I did not move it to `SimplifyParsers`, because other passes that are run here at the top-level also depend on that pass running first. Or do you mean running it before `SimplifyParsers` _in addition_ to running it before `MoveDeclarations`?

Either way, I will add a note inside of `SimplifyParsers` for now.